src/main/java/myGames/Game2.java

```diff
@@ -0,0 +1,60 @@
+package myGames;
+
+import static myGames.Utils.chooseItem;
+
+public class Game2 {
+
+
+    private static final String STR1 = "playerOne is WINNER!!!";
+    private static final String STR2 = "playerTwo is WINNER!!!";
+    private static final String STR3 = "no one's!!!";
+
+    static String myGame2(String playerOne, String playerTwo) {
+        String result = null;
+
+        System.out.println("The " + playerOne + " makes a move");
+        String itemOne = chooseItem();
+        System.out.println("The " + playerTwo + " makes a move");
+        String itemTwo = chooseItem();
+
+        if (itemOne.equals("Stone") && itemTwo.equals("Scissors")) {
```

Comment:
I think this big if-else can be optimized with some Map))

Reply:
Ok, thanks, something to think about))